Gemfile

```diff
@@ -12,15 +12,15 @@ gem 'pg', '~> 0.18'
 gem 'puma', '~> 3.7'
 gem 'rails', '~> 5.1.4'
 gem 'reek'
-gem 'rubocop', '0.46.0'
+gem 'rubocop', '0.48.1'
 gem 'rubocop-rspec', '1.8.0'
 gem 'sass-rails', '~> 5.0'
 gem 'turbolinks', '~> 5'
 gem 'uglifier', '>= 1.3.0'
 
 group :development, :test do
   gem 'byebug', platforms: %i(mri mingw x64_mingw)
-  gem 'capybara', '~> 2.13'
+  gem 'capybara', '~> 2.16', '>= 2.16.1'
```

Comment:
`~> 2.16` is enough here. What It basically does is that it takes the most recent `2.16.x` version, so let's say that we have such versions available:

- `2.16.1`
- `2.16.2`

It will take the second one.